### DISCUSSION TITLE
Installing list

### DISCUSSION THREAD

**Discussion by AdmixxTTP** [Romanitho/Winget-AutoUpdate#618] [Q&A]
Hello everyone,

I have a question:
Is it possible to create an installation list?
I've created and saved a list using the WAU Configurator.

WinGet.txt:
Adobe.Acrobat.Reader.64-bit
VideoLAN.VLC
Notepad++.Notepad++

My question now is, if I install Winget-AutoUpdate (deploy), how do I give it this list so it installs these right away?
Is that possible?
Do I need to use "Mods" for that?
If yes, how do I do that? I unfortunately didn't understand it.

Br
Admixx

**Comment by AndrewDemski-ad-gmail-com** [comment]
Hey @AdmixxTTP,

starting point for WAU action is building delta using two collections
- your list of currently recognized by winget applications
- current list of applications in winget repository

If those two collections are identical, WAU would not do a single thing.
There is no logic in WAU which would follow "install if missing" method.

If pre-/post- customization is not a factor for you, then I recommend calling simple "winget install --id (..)" commands.
